src/app/forms/custom-sections/required-standards/required-standards.component.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  Component, EventEmitter, Input, OnDestroy, OnInit, Output,
+} from '@angular/core';
+import { DynamicFormService } from '@forms/services/dynamic-form.service';
+import { CustomFormArray, CustomFormGroup, FormNode } from '@forms/services/dynamic-form.types';
+import { TestResultRequiredStandard } from '@models/test-results/test-result-required-standard.model';
+import { TestResultModel } from '@models/test-results/test-result.model';
+import { Subscription, debounceTime } from 'rxjs';
+
+@Component({
+  selector: 'app-required-standards[template]',
+  templateUrl: './required-standards.component.html',
+})
+export class RequiredStandardsComponent implements OnInit, OnDestroy {
+  @Input() isEditing = false;
+  @Input() template!: FormNode;
+  @Input() requiredStandards?: any[] | null = []; // TODO: fix this to use actual RS
```

Comment:
Is this being handled in another ticket or was this missed. Would be nice to have a properly type array.

Reply:
Yeah another ticket, will make sure it gets done proper :) it's because the actual method is returning defects still, so until that is fixed, no point typing it